entrypoints/apply-shim.js

```diff
@@ -211,7 +211,8 @@ if (!window.ShadyCSS || !window.ShadyCSS.ScopingShim) {
 
     nativeCss: nativeCssVariables,
     nativeShadow: nativeShadow,
-    cssBuild: cssBuild
+    cssBuild: cssBuild,
```

Comment:
Why not just
```
cssBuild,
disableRuntime
```

Reply:
Closure Compiler didn't like the shorthand